data/yaml/venues.yaml

```diff
@@ -1002,6 +1002,8 @@ stylevar:
   acronym: Style-Var
   is_acl: true
   name: Workshop on Stylistic Variation
+syntaxfest:
+  name: Joint Conference on Syntactic Research
```

Comment:
@mjpost not sure what the name for syntaxfest, I couldn't find a proper name on its website https://syntaxfest.github.io/, also not sure what value to set for `acronym`, `is_acl` and `is_toplevel` slots

Reply:
- acronym: SyntaxFest
- is_acl: False
- is_toplevel: False
- name: Joint meeting of four workshops on syntax